server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/eml/twoAttachments.eml

```diff
@@ -18,13 +18,13 @@ Date: Fri, 27 Mar 2015 21:48:38 +0100
 Message-Id: <20150227203838.0011.CanonTxNo.4037@canondir.par.lng>
 Mime-Version: 1.0
 Content-Type: multipart/mixed;
- boundary="AHNPACBLDCDIDAGGGDDFAABECGCA"
-
+ boundary="AHNPACBLDCDIDAGGGDDFAABECGCA"
+

```

Comment:
I lost at least two hour thinking there were a bug exploring a lot of code etc.. etc..
But no, this is because here we had \n\n instead \r\n\r\n